experiment/selinux/src/selinux.rs

```diff
@@ -141,8 +353,8 @@ impl SELinux {
     // function similar with setFSCreateLabel in go-selinux repo.
     // set_fscreate_label sets the default label the kernel which the kernel is using
     // for file system objects.
-    pub fn set_fscreate_label(&self, label: &str) -> Result<(), SELinuxError> {
-        return Self::write_con(self.attr_path("fscreate").as_path(), label);
+    pub fn set_fscreate_label(&mut self, label: &str) -> Result<usize, SELinuxError> {
```

Comment:
Can't we make labels more user-friendly by making them their own structure instead of `&str`?

Reply:
And, I think `experiment/selinux/src/selinux.rs` is too big a file, so It'd be better to separate the functions related to `label` to reduce `experiment/selinux/src/selinux.rs`

Reply:
> And, I think experiment/selinux/src/selinux.rs is too big a file, so It'd be better to separate the functions related to label to reduce experiment/selinux/src/selinux.rs

Agree. I divided selinux.rs into two files, sharing same impl `SELinux`. Is this what you meant ?

> Can't we make labels more user-friendly by making them their own structure instead of &str?

Sounds very reasonable, but I am not sure how I should define new structure for labels.
Could you explain a little bit more about your idea on this?
You mean like this?

```
struct SELinuxLabel { aaa }
impl SELinuxLabel {fn ...}
```

If so, I don't think it is a good idea because a label should be passed as an arg for this struct and impl.
Therefore I would like to know your idea before starting the implementation :)